calicoctl/commands/ipam/show.go

```diff
@@ -69,7 +69,8 @@ Description:
 	// `IP 192.168.71.1 is not assigned in block`. This is not exactly an error,
 	// so not returning it to the caller.
 	if err != nil {
-		return err
+		fmt.Println(err)
```

Comment:
use stderr?

Reply:
I stick with pure revert atm

Reply:
Yeah, this isn't an error case so I think stdout is probably right,